content/en/news/2019/announcing-1.4/_index.md

```diff
@@ -11,4 +11,49 @@ aliases:
 
 We are pleased to announce the release of Istio 1.4!
 
+Version 1.4 continues our efforts at improving the user experience in Istio,
+with a focus on simplification. We've also continued to add features that
+improve the performance and experience of running Istio.
+
+## Mixer-less telemetry
+Our implementation of telemetry without Mixer will simplify installation and
+operation of meshes, all while vastly improving performance. The in-proxy
```

Comment:
Do we want to be adding links to docs for these things so folks can get more details on these? Not sure if we prefer to just keep this high level and have the release notes handle that.

Reply:
We didn't last time -- I just followed the convention set there. General descriptions (without links) of highlights at the top, followed by release notes with links.